tools/app_interface_reporter.py

```diff
@@ -225,25 +218,31 @@ def get_apps_data(date, month_delta=1, thread_pool_size=10):
                            auth=(dashdotdb_user, dashdotdb_pass)).text
     namespaces = queries.get_namespaces()
 
-    saas_deploy_jobs = {}
+    build_jobs = jjb.get_all_jobs(job_types=['build'])
+    jobs_to_get = build_jobs
+
+    saas_deploy_jobs = []
     for saas_file in saas_files:
         saas_file_name = saas_file['name']
         for template in saas_file["resourceTemplates"]:
             for target in template["targets"]:
                 job = {}
-                job['branch'] = target["namespace"]["environment"]["name"]
+                job['env'] = target["namespace"]["environment"]["name"]
                 job['name'] = get_openshift_saas_deploy_job_name(
-                    saas_file_name, job['branch'], settings
+                    saas_file_name, job['env'], settings
                 )
+                job['saas_file_name'] = saas_file_name
                 job['instance'] = saas_file["instance"]["name"]
                 job['app'] = saas_file["app"]["name"]
-                if job['instance'] not in saas_deploy_jobs:
-                    saas_deploy_jobs[job['instance']] = [job]
+                saas_deploy_jobs.append(job)
+                if job['instance'] not in jobs_to_get:
+                    jobs_to_get[job['instance']] = [job]
```

Comment:
You are also modifying `build_jobs`. Is that what you want?

Reply:
Ahh nice catch!! < 3 I've updated `jobs_to_get` to be `jobs_to_get = build_jobs.copy()`so that line 238 - 241 will not modify `build_jobs`.